openstudiocore/src/model/CoilCoolingDXMultiSpeed.cpp

```diff
@@ -303,6 +308,29 @@ namespace detail {
     return false;
   }
 
+  boost::optional<AirflowNetworkEquivalentDuct> CoilCoolingDXMultiSpeed_Impl::createAirflowNetworkEquivalentDuct(double length, double diameter)
```

Comment:
@kbenne this method could be added to a Coil base class if there was one

Reply:
Possibly any StraightComponent could have an AirflowNetworkEquivalentDuct? @jasondegraw ?